tests/rkt_net_test.go

```diff
@@ -608,6 +608,10 @@ func testNetCustomNatConnectivity(t *testing.T, nt networkTemplateT) {
 
 	// Child connects to host
 	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatalf("os.Hostname() got error %q", err)
```

Comment:
I generally like to distinguish failures in test behaviour (errorf, fatalf) from failures in setup by having the latter panic. Does that distinction make sense?